### ISSUE TITLE
Is this supposed to remove all jokers?

### ISSUE THREAD

**Issue by achar45** [AkitaAttribute/ReplaceJokers#2]
Thank you for your joker cap mod!...I've been having fun with it.

When I install this one it simply removes all the jokers from my collection and from the shop is this the intended effect? Not sure how this is supposed to work.

Do you think it's possible to make a mod that increases the rarity of the soul card to get legendary jokers?

**Comment by AkitaAttribute** [comment]
Yeah that's kind of the intended effect.  The intention is to add them back via the json file, but it doesn't work as well as I wanted it to.  There is still a bit of jank in adding the jokers back sadly.  I was playing around with it more last week, but couldn't round it out.

I can look into the soul card probability.

**Comment by AkitaAttribute** [comment]
@achar45 

https://github.com/AkitaAttribute/IncreaseSoulProbability

**Comment by achar45** [comment]
Awesome! Thank you

**Comment by zjwilde** [comment]
I'm not sure if this helps, but in the paperback mod, there's a "minor arcana card" called the "Nine of Swords" which says "Destroy selected Joker It is unable to appear again for the remainder of the run" 

I'm not sure how they did it because in the code it calls a function, and I'm not sure where it comes from, but maybe you can find it. Or possibly the people who did that mod know how to do what you're trying to do.
